src/test/url-handling.test.js

```diff
@@ -340,6 +341,30 @@ describe('search strings', function() {
   });
 });
 
+describe('profileName', function() {
+  it('serializes the profileName in the URL ', function() {
+    const { getState, dispatch } = _getStoreWithURL();
+    const profileName = 'Good Profile';
+
+    dispatch(changeProfileName(profileName));
+    const urlState = urlStateReducers.getUrlState(getState());
+    const { query } = urlStateToUrlObject(urlState);
+    expect(query.profileName).toBe(profileName);
+  });
+
+  it('reflects in the state from URL', function() {
+    const { getState } = _getStoreWithURL({
+      search: '?profileName=XXX',
+    });
+    expect(urlStateReducers.getProfileNameFromUrl(getState())).toBe('XXX');
```

Comment:
Can you please add a check for `getProfileName` as well? As ths is the one we use in the end :)
(same below please)

Reply:
Hi @julienw, Thanks for the patch. I have made the changes as you suggested. Please have a look and suggest if there are more changes need to be done :)